src/plugins/timeline/Plan.vue

```diff
@@ -85,7 +85,10 @@ export default {
         },
         updateViewBounds() {
             this.viewBounds = this.openmct.time.bounds();
-            // this.viewBounds.end = this.viewBounds.end + (30 * 60 * 1000);
+            //Add a 30 min padding to the end bounds to look ahead
+            let endDate = new Date(this.viewBounds.end);
+            endDate.setMinutes(endDate.getMinutes() + 30);
```

Comment:
I thought we were going to add a 50% margin here? I might be misremembering though. A percentage margin is a little more compatible with non-UTC time systems (which we don't need to support right now, but also don't want to close the door to them prematurely).

Also, I don't think `setMinutes` works like this, it only accepts minutes values between 0-59. So, there is a potential overflow problem here.

Another approach to this might be something like:
```
let timespan = (this.viewBounds.end - this.viewBounds.start);
let padding = timespan * 0.5;
this.viewBounds.end = timespan + padding;
```

Reply:
Duh! of course...not sure what I was thinking here.
Hmm I thought we were going to look ahead by a specific duration. I'm fine with a percentage margin too. I'll fix it.